contracts/Transition.sol

```diff
@@ -340,7 +337,7 @@ contract Transition is TellorStorage, TellorVars {
      * (or disputes on old Tellor values)
      */
     fallback() external {
-        address addr = 0xdDB59729045d2292eeb8Ff96c46B8db53B88Daa2; // Main Tellor address (Harcode this in?)
+        address addr = 0x2754da26f634E04b26c4deCD27b3eb144Cf40582; // Main Tellor address (Harcode this in?)
```

Comment:
If you look at the [master contract on etherscan](https://etherscan.io/address/0x88dF592F8eb5D7Bd38bFeF7dEb0fBc02cf3778a0#readProxyContract), it tells you the [address for the implementation contract](https://etherscan.io/address/0x2754da26f634e04b26c4decd27b3eb144cf40582), as well as the [address for the old implementation contract](https://etherscan.io/address/0xddb59729045d2292eeb8ff96c46b8db53b88daa2). Etherscan says the current address is 0x2754d. Also the 0xdDB59 contract is a few months older than the 0x2754d contract. Tests don't fail for either address, and I believe that's the case because they both contain most of the same logic, and any storage being operated on resides at the master address.